pkg/chartutil/load.go

```diff
@@ -255,7 +255,13 @@ func LoadFile(name string) (*chart.Chart, error) {
 	}
 	defer raw.Close()
 
-	return LoadArchive(raw)
+	c, err := LoadArchive(raw)
+	if err != nil {
+		if err.Error() == "gzip: invalid header" {
```

Comment:
I'm not sure we can safely assume that the error `gzip: invalid header` means we are reading from an uncompressed file. We could also be reading a corrupt gzipped archive.

Is there some way you could try to improve the error checking either here or in `LoadArchive`?